### PR TITLE
Added BrushUsersCounter

### PR DESCRIPTION
**Pull request by KevinDaGame**
This counter will keep track of how many players that are online actually use VoxelSniper

### CODE REVIEW

File: VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.github.kevindagame.voxelsniper.bstats
+
+import com.github.kevindagame.voxelsniper.events.player.PlayerSnipeEvent
+import java.util.*
+
+class BrushUsersCounter {
+    fun registerListeners() {
+        PlayerSnipeEvent.registerListener { event: PlayerSnipeEvent -> onBrushUse(event) }
```
Comment (Lennart99) [inline VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]:
```suggestion
        PlayerSnipeEvent.registerListener(this::onBrushUse)
```

**Review by Lennart99 — APPROVED** [review]

File: VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt (hunk 1 above)
Comment (KevinDaGame) [reply VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]: Die change vind kotlin niet leuk. Ik laat het zo

Comment (Lennart99) [reply VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]: seemed to work fine for me..

Comment (KevinDaGame) [reply VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]: image

Comment (KevinDaGame) [reply VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]: nvm zie al wat er mis is

Comment (Lennart99) [reply VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt:8]: haakjes

**Pull request merged**